### PR TITLE
documentation(CONTRIBUTING.md) add rntester info to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by lbochenek**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -21,3 +21,25 @@ The build tools for v140 (Platform Toolset = 'v140') cannot be found. To build u
 Retarget the solution to v141:
 1. Right click the ReactNative solution and click "Retarget Solution"
 2. Make sure the Platform Toolset has Upgrade to v141 selected, and click "OK."
+
+
+## Using RNTester
+RNTester is a React Native Windows App that demonstrates the implemented views and modules of React Native Windows.
+
+You can use it to test your changes to react-native-windows by making sure your changes haven't broken the views and modules.
```
Comment (rozele) [inline CONTRIBUTING.md]: Inconsistent use of `react-native-windows` and `React Native Windows` in this section.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -21,3 +21,25 @@ The build tools for v140 (Platform Toolset = 'v140') cannot be found. To build u
...
+
+#### But How?
+Before starting make sure you have run `npm install` in the react-native-windows directory. Additionally, make sure the RNTester submodule is up to date by running `git pull --recursive-submodules` from the react-native-windows directory.
+
+Use Visual Studio 2015 or higher, with the Windows 10 SDK 10.0.10586 or higher.
```
Comment (rozele) [inline CONTRIBUTING.md]: Should this be 10.0.14393 at this point?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -21,3 +21,25 @@ The build tools for v140 (Platform Toolset = 'v140') cannot be found. To build u
...
+
+1. Open the RNTester solution file (react-native-windows/RNTester.sln) in Visual Studio
```
Comment (rozele) [inline CONTRIBUTING.md]: react-native-windows/RNTester/RNTester.sln

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -21,3 +21,25 @@ The build tools for v140 (Platform Toolset = 'v140') cannot be found. To build u
...
+2. Set RNTesterApp as the StartUp project
+3. Set the Solution Configuration to "Debug" and the Solution Platforms to "x86"
+4. Run the project on the local machine by clicking the run button or by pressing F5
+
+A Windows app will open with a red error screen stating `Unable to download JS bundle. Did you forget to start the development server or connect your device?` We still need to start the dev server!
```
Comment (rozele) [inline CONTRIBUTING.md]: may open with a red box dialog

Comment (rozele) [reply CONTRIBUTING.md]: Maybe break this section out about starting the packager server into its own doc / FAQ (if it's not already documented somewhere). Perhaps we should add `npm start` options to run `react-native start` as well.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -21,3 +21,25 @@ The build tools for v140 (Platform Toolset = 'v140') cannot be found. To build u
...
+5. In your command line, make sure that you are in the react-native-windows directory and run then `react-native start`
+6. Reload the bundle in the app by clicking "Reload JavaScript" or by pressing Ctrl+R
+
+The dev server will send over a bundle, and you can now use RNTester to test your changes to the repository.
```
Comment (rozele) [inline CONTRIBUTING.md]: The packager server will bundle the application JavaScript

File: CONTRIBUTING.md (hunk 2 above)
Comment (lbochenek) [reply CONTRIBUTING.md]: I honestly don't know, I got that from the FB RNTester readme. You're probably right, I'll change it

File: CONTRIBUTING.md (hunk 4 above)
Comment (lbochenek) [reply CONTRIBUTING.md]: starting the packager server is talked about in the README when discussing just running a regular rnw app. Maybe keep this part in here for now, and then later when I write more about getting started I'll break it out into it's own section?

Comment (rozele) [reply CONTRIBUTING.md]: that's fine. just change "will open with a red error screen" to "may open with a red box dialog"

File: CONTRIBUTING.md (hunk 2 above)
Comment (rozele) [reply CONTRIBUTING.md]: FB RNTester README is wrong as well. Everything should be at 10.0.14393 at this point.

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -24,22 +24,22 @@ Retarget the solution to v141:
...
+Use Visual Studio 2015 or higher, with the Windows 10 SDK 10.0.14393 or higher.
 
-1. Open the RNTester solution file (react-native-windows/RNTester.sln) in Visual Studio
+1. Open the RNTester solution file (react-native-windows/RNTester/RNTester.sln) in Visual Studio
 2. Set RNTesterApp as the StartUp project
 3. Set the Solution Configuration to "Debug" and the Solution Platforms to "x86"
 4. Run the project on the local machine by clicking the run button or by pressing F5
 
-A Windows app will open with a red error screen stating `Unable to download JS bundle. Did you forget to start the development server or connect your device?` We still need to start the dev server!
+A Windows app will open, and it may have a red box dialogue stating `Unable to download JS bundle. Did you forget to start the development server or connect your device?` We still need to start the dev server!
```
Comment (rozele) [inline CONTRIBUTING.md]: dialog spelled as in Dialog API https://docs.microsoft.com/en-us/uwp/api/Windows.UI.Popups.MessageDialog

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -24,22 +24,22 @@ Retarget the solution to v141:
 
 
 ## Using RNTester
-RNTester is a React Native Windows App that demonstrates the implemented views and modules of React Native Windows.
+RNTester is a React Native Windows app that demonstrates the implemented views and modules of React Native Windows.
 
-You can use it to test your changes to react-native-windows by making sure your changes haven't broken the views and modules.
+You can use it to test your changes to React Native Windows by making sure your changes haven't broken the views and modules.
```
Comment (rozele) [inline CONTRIBUTING.md]: Don't need to fix it here, but we have inconsistencies throughout the documentation of React Native Windows vs. react-native-windows. Honestly, I think I use both when writing about the project, so it probably doesn't matter.

Comment (lbochenek) [reply CONTRIBUTING.md]: yeah I kinda view it as React Native Windows referring to the project as a whole, and react-native-windows as referring to the code. idk where I got that or if other people are using it like that.

**Pull request merged**